### PR TITLE
rmi, rm: add test

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: Zhou Hao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

**Comment by ripcurld** [comment]
@rhatdan @TomSweeneyRedHat 
Although `bats` makes it easy to write tests, I prefer writing unit-tests in Go because we can have code coverage and it motivates you to write unit-tests which give confidence to change things in the future.

I know we have unit-tests but looking at the code base it looks like we have more `bats` tests than unit-tests, doesn't it?

**Comment by pixdrift** [comment]
I've seen ways to get coverage for external testing tools for go, it would be awesome to get coverage for bats tests.

File: tests/rmi.bats [hunk 1]
```diff
@@ -27,6 +27,15 @@ load helpers
   [ "$output" == "" ]
 }
 
+@test "remove multiple images errors" {
```
Comment (TomSweeneyRedHat) [inline tests/rmi.bats]: suggest changing "images" to "non-existent images"

**Comment by TomSweeneyRedHat** [comment]
@q384566678 thanks for the PR.  I've one very small suggestion request for a name in one of your tests.  If you could do that and push again, I think the Travis CI test issue will clear itself up as it looks to be a hiccup/flake.

As far as the type of test go, @nalind started the test suite using bats and it grew over time to what it is now.  Since then other folks have added *.go tests here and there.  Longer term I'd like to convert the tests to ginkgo based bdd tests.  They're written in go and should allow for coverage.  That and getting initial code coverage going with CodeCov has been a "background" project of mine that I keep making very slow progress on.  #516 is my current attempt for Buildah.  My main issue there is getting Buildah to run inside of the Travis CI Ubuntu so the ginkgo tests can run.  Podman does most of their testing using gingko having converted most of if not all of their bats, but they use a different test environment spinning up a container to do it within so they've not run into the Ubuntu fun there.  

So in short, we're trying to keep stuff in bats for now for continuity and once we get ginkgo setup, then we can start converting from bats and not have to search all over for the files.

File: tests/rmi.bats (hunk 1 above)
Comment (q384566678) [reply tests/rmi.bats]: updated， PTAL.
